pages/1.10/security/ent/tls-ssl/ca-custom/index.md

```diff
@@ -8,26 +8,322 @@ excerpt:
+
+- on the master nodes:
+    - `/var/lib/dcos/pki/tls/CA/private/custom_ca.key` file containing the private key associated with the custom CA certificate. The file is owned by the root Unix user and has 0600 permissions set
+
+Here is an example of the `issuer` and `subject` fields of a custom root CA certificate:
+
+```bash
+cd $DCOS_INSTALL_DIR
+openssl x509 -in genconf/dcos-ca-certificate.crt -issuer -subject -noout
+```
+```bash
+issuer= /DC=io/DC=integration-test/C=DE/ST=Utopia/O=DC/OS/OU=Brogrammer Unit/CN=Integration Test Root CA
+subject= /DC=io/DC=integration-test/C=DE/ST=Utopia/O=DC/OS/OU=Brogrammer Unit/CN=Integration Test Root CA
+```
+
+Since the custom CA certificate is a root CA certificate and the corresponding CA certificate chain is empty, we have to omit the ca_certificate_chain_path parameter in the configuration file. The corresponding configuration parameters have to be specified as follows in the `$DCOS_INSTALL_DIR/genconf/config.yaml` file on the bootstrap node:
+
+```bash
+ca_certificate_path: genconf/dcos-ca-certificate.crt
+ca_certificate_key_path: genconf/dcos-ca-certificate-key.key
+```
+
+## Use case 2: The custom CA certificate is an intermediate CA certificate, directly issued by a root CA
+
+In this case the custom CA certificate is an intermediate one, issued directly by a root CA. The CA certificate chain consists of just that root CA certificate.
+
+The following files are present:
+
+- on the bootstrap node:
+    - `$DCOS_INSTALL_DIR/genconf/dcos-ca-certificate.crt` file containing the custom CA certificate in PEM format
+    - `$DCOS_INSTALL_DIR/genconf/dcos-ca-certificate-key.key` file containing the private key associated with the custom CA certificate in the PKCS#8 format
+    - `$DCOS_INSTALL_DIR/genconf/dcos-ca-certificate-chain.crt` file containing the certificate chain in PEM format
+
+- on the master nodes
+    - `/var/lib/dcos/pki/tls/CA/private/custom_ca.key` file containing the private key associated with the custom CA certificate in the PKCS#8 format. The file is owned by the root Unix user and has 0600 permissions set
+
+Here is an example of an appropriate intermediate custom CA certificate: 
+
+```bash
+cd $DCOS_INSTALL_DIR
+openssl x509 -in genconf/dcos-ca-certificate.crt -issuer -subject -noout
+```
+```bash
+issuer= /DC=io/DC=integration-test/C=DE/ST=Utopia/O=DC/OS/OU=Brogrammer Unit/CN=Integration Test Root CA
+subject= /DC=io/DC=integration-test/C=DE/ST=Utopia/O=DC/OS/OU=Brogrammer Unit/CN=Integration Test Intermediate CA 01
+```
+ 
+Here is an example of a corresponding CA certificate chain:  
+
+```bash 
+cd $DCOS_INSTALL_DIR
+cat genconf/dcos-ca-certificate-chain.crt dcos-ca-certificate-chain.crt | awk -v cmd="openssl x509 -issuer -subject -noout && echo" '/-----BEGIN/ { c = $0; next } c { c = c "\n" $0 } /-----END/ { print c|cmd; close(cmd); c = 0 }'
```

Comment:
This shows a file path twice here: `cat genconf/dcos-ca-certificate-chain.crt dcos-ca-certificate-chain.crt`.